unfetter-discover/unfetter-store
# Issue 663

Fixes unfetter-discover/unfetter#663
Fixes unfetter-discover/unfetter#662

Added remaining base swagger-doc, controllers and tests for dodcar objects.
Also refactored `base-stix-routes.spec.js` to be more readable and to add dodcar tests.

## To Test:
Pull issue-663
Restart stack
Make sure app still works
Visit API page and ensure everything looks good
In a separate terminal run `docker exec -it unfetter-discover-api npm run test` and ensure all tests pass.
